_layouts/software-details.html

```diff
@@ -500,32 +500,63 @@ <h2>
               <div class="col-6 col-md">
                 <p>
                   <span class="label">{{ t.software.enabling_platforms }}</span>
-                  {% if page.publiccode.it.piattaforme == nil or page.publiccode.it.piattaforme.size == 0 %}
-                      Nessuna
+                  {% assign p = page.publiccode.it.piattaforme %}
+                  {% if p == nil or p.size == 0 or p.size == 4 and p.pagopa == false and p.spid == false and p.cie == false and p.anpr == false %}
+                      {{ t.software.dependencies_none }}
                   {% else %}
-                      {% if page.publiccode.it.piattaforme.pagopa == true %}
+                      {% if p.pagopa == true %}
                       <a href="/{{ active_lang }}/pagopa" class="enabling_platforms">
                         pagoPA
                       </a>
                       {% endif %}
-                      {% if page.publiccode.it.piattaforme.spid == true %}
+                      {% if p.spid == true %}
                       <a href="/{{ active_lang }}/spid" class="enabling_platforms">
                         SPID
                       </a>
                       {% endif %}
-                      {% if page.publiccode.it.piattaforme.cie == true %}
+                      {% if p.cie == true %}
                       <a href="/{{ active_lang }}/cie" class="enabling_platforms">
                         CIE
                       </a>
                       {% endif %}
-                      {% if page.publiccode.it.piattaforme.anpr == true %}
+                      {% if p.anpr == true %}
                       <a href="/{{ active_lang }}/anpr" class="enabling_platforms">
                         ANPR
                       </a>
                       {% endif %}
                    {% endif %}
                 </p>
               </div>
+              <div class="col-6 col-md">
+                <p>
+                  <span class="label">{{ t.software.compliance }}</span>
+                  {% assign p = page.publiccode.it.conforme %}
+                  {% if p == nil or p.size == 0 or p.size == 4 and p.gdpr == false and p.lineeGuidaDesign == false and p.misureMinimeSicurezza == false and p.modelloInteroperabilita == false %}
+                      {{ t.software.dependencies_none }}
+                  {% else %}
+                      {% if p.gdpr == true %}
+                      <a href="{{ t.software.gdpr_url }}" class="enabling_platforms">
```

Comment:
Should we keep this external links? I don't particularly like it, however I've inserted the links to see how it looks.

Reply:
Also, I kept the same class `enabling_platforms` since it's basicallly the same style as the other section.